src/dotnet-svcutil/lib/src/dotnet-svcutil-lib.csproj

```diff
@@ -65,7 +65,12 @@
     <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
     <PackageReference Include="System.Reflection.DispatchProxy" Version="4.4.0" />
   </ItemGroup>
-
+  
+  <Target Name="CopyPackageDlls" AfterTargets="build">
+    <ItemGroup>
+      <DesignData Include="$(OutputPath)internalAssets\**" Pack="true" PackagePath="internalAssets" />
```

Comment:
same as above.

Reply:
This needs to be packaged into the dotnet-svcutil-lib package for use in reflection.